nav2_planner/src/planner_server.cpp

```diff
@@ -79,26 +75,26 @@ PlannerServer::on_configure(const rclcpp_lifecycle::State & state)
 
   tf_ = costmap_ros_->getTfBuffer();
 
-  get_parameter("planner_plugin_ids", plugin_ids_);
-  get_parameter("planner_plugin_types", plugin_types_);
-  auto node = shared_from_this();
-
-  if (plugin_ids_.size() != plugin_types_.size()) {
-    RCLCPP_FATAL(
-      get_logger(),
-      "Planner plugin names and types sizes do not match!");
-    exit(-1);
+  get_parameter("plugins", planner_ids_);
```

Comment:
I'm on the fence about changing this to `plugins` from `planner_plugins`. On one hand, its nice that all of these now share a variable name for plugins, and on the other hand, it might not be clear what type of plugins these are if we have servers with multiple plugin interfaces (e.g. controller server will in future have controller, progress, and goal checker plugins)

Reply:
Makes sense. Then let's name the core plugin parameters `planner_plugins`, `controller_plugins`, and `recovery_plugins` and keep the costmap name just `plugins`?

Reply:
OK, lets do that. I think then the util will need to take in a string name for the plugins namespace (which is probably good anyhow)

Reply:
The renames were done in the latest commits, util already takes in the plugin name to find `plugin_name.plugin`.